### ISSUE TITLE
I should see project folders I am a member of

### ISSUE THREAD

**Issue by rickatech** [rickatech/backstage#8]
I have a Windows 10 setup with recent plug-in installed.

ProLoop user
  rickatech
  password

I can see the correct project folders with http://newui.proloop.com/dav , but when I try to open file and click ProLoop, the projects folder appears empty.

Related
  - See #7 
![gh_oc 5_0](https://cloud.githubusercontent.com/assets/11014263/25363410/0d16e17a-290f-11e7-86f3-4f9b6498c48f.png)
![gh_oc 5_1](https://cloud.githubusercontent.com/assets/11014263/25363411/108efe32-290f-11e7-8634-11e75e95ea11.png)

**Comment by rickatech** [comment]
This may be due to some backend enhancements that restrict which folders under projects master folder users can see.  This new functionality is needed to protect file access against users who are not members of certain projects.

**Comment by sdn-dev** [comment]
We stored the username and password in the registry. Earlier, we used an account "sn" to test the functionality. This account doesn't have permissions to access Project folders. Due to the recent changes in the backend, no projects folders will show up for this user. If we update the username and password in the registry to rickatech and password, then folders will show up.

To verify the permission-based folder access, we can map a drive using the "net use w: http://newui.proloop.com/dav/ /user:rickatech password" and open the "W:\", we'll see the project folders: Chicago and RickTest under Projects.
